src/main/java/org/embulk/filter/column/ColumnVisitorImpl.java

```diff
@@ -244,13 +245,19 @@ public void jsonColumn(Column outputColumn)
                 pageBuilder.setNull(outputColumn);
             }
             else {
-                String jsonPath = new StringBuilder("$.").append(outputColumn.getName()).toString();
+                String name = outputColumn.getName();
+                // Strips any of a set of characters from the start and end of a String.
```

Comment:
I mistook twice. 

String literal of the Java is in the double quotes. So, single quotes doesn't need escape. 
 And StringUtils.strip needs take a result to variables. So, the changes in earlier was meaningless.
 
So, I will remove the StringUtils.strip
And I will add more test case.